WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -126,12 +126,21 @@ def do_import
                        .map(&:to_hash)
                        .reject { |registration| registration.values.all?(&:nil?) }
                        .select { |registration| registration[:status] == "a" }
+    if competition.competitor_limit_enabled? && registrations.length > competition.competitor_limit
+      raise "The given file includes #{registrations.length} accepted #{"registration".pluralize(registrations.length)}"\
+            ", while #{competition.competitor_limit} is the competitor limit."
```

Comment:
nit: Maybe clearer to say ", which is more than the competitor limit of #{....}."?